doc/ReadingGuide.md

```diff
@@ -1,4 +1,32 @@
-# A guide to reading the Sail RISC-V specification
+# Using the Sail RISC-V specification
+
+The most common workflows using the Sail model of RISC-V are shown
+below.
+
+![](./figs/workflow.svg)
+
+The Sail components of the model are a Sail project file
+(`riscv.sail_project`) and several associated files of Sail source
+code. The Sail compiler translates these components into a C++ model
+(`sail_riscv_model.{h,cpp}`), which is then wrapped in a C++ harness
+and compiled into a `sail_riscv_sim` executable binary to give an
+RISC-V emulator.
+
+When running this emulator on a RISC-V ELF binary (`test.elf`), the
+emulator can also be a provided a configuration file in JSON format
+(named `dut_config.json` above); if no configuration file is provided,
+a default RV64 configuration is assumed. The Sail compiler also
+generates a JSON schema (`sail_riscv_config_schema.json`) for the
+configuration file from the Sail sources; every configuration file is
+validated against this schema before use. More information on using
+the emulator is available using its `-h` help command-line option.
+
+The Sail model can also be used to generate `JSON` and `HTML`
+artifacts for documentation. A prototype of their use to annotate the
+unprivileged volume of the RISC-V specification is available
+[here](https://github.com/Timmmm/riscv-isa-manual).
```

Comment:
@Timmmm I linked to your github repo of the manual instead of the website, to save bandwidth from the pre-commit checks in CI.

Reply:
The website is hosted on Github Pages anyway so I don't think that will make any difference... But I think this is fine anyway.